NuRadioReco/modules/RNO_G/hardwareResponseIncorporator.py

```diff
@@ -76,15 +84,18 @@ def get_filter(self, frequencies, station_id, channel_id, det,
             Note: The adjustment to the minimal gain is NOT visible when getting the amp response from
             ``analog_components.get_amplifier_response()``
 
+        is_trigger_channel: bool
+            Use trigger channel response instead. Only relevant for RNO-G. (Default: False)
```

Comment:
Doesn't match the parameter name. Is it worth just using `trigger` to agree with the name in the detector module?